optuna/study/study.py

```diff
@@ -307,6 +307,9 @@ def optimize(
         default choice for the sampler is TPE.
         See also :class:`~optuna.samplers.TPESampler` for more details on 'TPE'.
 
+        An optimization will be stopped when receiving a termination signal such as SIGINT and SIGTERM.
```

Comment:
https://github.com/optuna/optuna/runs/5562609091?check_suite_focus=true
This line is too long. Please fix and check if CI pass. :pray: